sys/kern/kern_descrip.c

```diff
@@ -2424,6 +2450,7 @@ fdcopy(struct filedesc *fdp)
 	newfdp->fd_freefile = fdp->fd_freefile;
 	FILEDESC_FOREACH_FDE(fdp, i, ofde) {
 		if ((ofde->fde_file->f_ops->fo_flags & DFLAG_PASSABLE) == 0 ||
+		    ((ofde->fde_flags & UF_FOCLOSE) != 0) ||
```

Comment:
Extra parens, this can just be `(ofde->fde_flags & UF_FOCLOSE) != 0`, just like the previous line.